test/integration/adapters.js

```diff
@@ -15,7 +15,7 @@ var collectedData = {}
  * @type {Array}
  * @const
  */
-const adapters = ['Jasmine', 'QUnit']
+const adapters = ['Jasmine', 'QUnit', 'Mocha']
```

Comment:
Sure. But I think we should change this in another issue/branch, where we are refactoring testing, because there will be one, for adding more tests, use the same `referenceData obj`.